src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -2234,6 +2234,7 @@ void Pet::InitTalentForLevel()
 uint8 Pet::GetMaxTalentPointsForLevel(uint8 level)
 {
     uint8 points = (level >= 20) ? ((level - 16) / 4) : 0;
+    points = uint8(points * sWorld->getRate(RATE_TALENT_PET));
```

Comment:
move this to L#2244 at the return

Reply:
I just want to change the talent points brought by level, and don't want to affect other things. Maybe this is more in line with expectations, what do you think?

Reply:
> move this to L#2244 at the return

I just want to change the talent points brought by level, and don't want to affect other things. Maybe this is more in line with expectations, what do you think?

Reply:
The way it is worded it should modify the rate of the talent points, which would be at the end when all talent point calculations are done.

See how the config works for player talents, it is also calculated at the end

Reply:
> The way it is worded it should modify the rate of the talent points, which would be at the end when all talent point calculations are done.
> 
> See how the config works for player talents, it is also calculated at the end

yep, you are right